enterprise/api/license.go

```diff
@@ -1,17 +1,62 @@
 package api
 
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/bytebase/bytebase/api"
+)
+
+// validPlans is a string array of valid plan types.
+var validPlans = []string{
```

Comment:
Should this be []PlanType instead to avoid double conversion below?

Reply:
Updated. The string here is to make log validPlans more simple by using `strings.Join`. I updated the log now.